src/main/java/Duke.java

```diff
@@ -6,5 +9,78 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        String intro = "Hello! I'm Duke \n" +
+                    "What can I do for you? \n";
+        System.out.println(intro);
+        List<Task> ls = new ArrayList<>();
+        while(true) {
+            try { 
+                Scanner sc = new Scanner(System.in);
+                Task n = new Task(sc.nextLine());
+                if(n.description.equals("bye")) {
+                    break;
+                } else if (n.description.equals("list")) {
+                    System.out.println("Here are the things you've tried to do\n");
+                    for(int i = 1; i <= ls.size(); i++) {
+                        System.out.println(i + " " + ls.get(i -1));
+                    }
+                } else if(n.description.startsWith("mark")) {
+                    int x = Integer.parseInt(n.description.substring(5));
+                    Task edited = ls.remove(x -1);
+                    edited.done();
+                    ls.add(x-1, edited);
+                    for(int i = 1; i <= ls.size(); i++) {
+                        System.out.println(i + " " + ls.get(i -1));
+                    }
+                } else if (n.description.startsWith("unmark")) {
+                    int x = Integer.parseInt(n.description.substring(7));
+                    Task edited = ls.remove(x -1);
+                    edited.undone();
+                    ls.add(x-1, edited);
+                    for(int i = 1; i <= ls.size(); i++) {
+                        System.out.println(i + " " + ls.get(i -1));
+                    }
+                } else if(n.description.startsWith("todo")) {
+                    todo a = new todo(n.description.substring(5));
```

Comment:
```suggestion
                    todo newTask = new todo(n.description.substring(5));
```
For coding standard, to give more meaningful variable names